scripts/averaging/create_climo_files.py

```diff
@@ -8,12 +8,39 @@ def my_formatwarning(msg, *args, **kwargs):
     return str(msg) + '\n'
 warnings.formatwarning = my_formatwarning
 
+from uuid import int_
+from mypyc.ir.rtypes import optional_value_type
+from operator import truediv
```

Comment:
I think these got added erroneously, I'm guessing by autocomplete in VS Code?

Reply:
Yeah, I'm not sure how those got in there, but I've removed them.